wiki/markdown/configuration.md

```diff
@@ -168,4 +168,57 @@ and the settings are defined in the **ldap.server.list** property.
             "password": "",\
             "user.filter": "uid={0}"\
         }\
-    ]\
\ No newline at end of file
+    ]\
+
+## Password encryption
+
+In a production environment and filesystems, administrator most likely don't want to store passwords in clear text.
+
+These Eiffel-Intelligence password properties can be provided encrypted:
+        
+        rabbitmq.password
+        spring.data.mongodb.password
+
+
+A password property is configured by providing the encrypted password in this format.
+
+        some.component.password=ENC(<encrypted password>)
```

Comment:
Does mvn site like the < signs? Have you checked locally if the mvn site goal can render HTML pages from this file? If not, replace the < with { characters instead.

Reply:
Solved.